src/node/desktop/src/core/file-path.ts

```diff
@@ -35,18 +35,18 @@ const homePathAlias = '~/';
 const homePathLeafAlias = '~';
 
 /**
- * Removes duplicate separators from a path, given a separator for search.
+ * Normalize separators of a given path.
  *
  * @param {string} path
  * @param {string} [separator='/']
  * @return {*} 
  */
 function normalizeSeparators(path: string, separator = '/') {
-  return path.replace(/[\\/]+/g, separator);
+  return path.replace(/[\\]/g, separator);
```

Comment:
What happens if you run `normalizeSeparators("a/b/c", "\\")`?

Reply:
Fixed on [c61c6d5](https://github.com/rstudio/rstudio/pull/11311/commits/c61c6d56ed0b2be7d9227cbb91f664c6502e4452)